pkg/arvo/sys/hoon.hoon

```diff
@@ -11465,6 +11482,16 @@
   =/  gun  (~(mint ut p:$:tap) %noun gen)
   |.  ~+
   [p.gun .*(q:$:tap q.gun)]
```

Comment:
This is a preexisting condition, but the `|.` here should also get a minimal subject with `=>`.